test/backends/subdomain_services.spec.ts

```diff
@@ -20,6 +20,12 @@ const defs: any[] = [
   { backend: squarespace, tests: [
     { subdomain: "archmotorcycle" },// whoah
     { subdomain: "archmotorcycle", hostname: "www.archmotorcycle.com" } // whoah
+  ]},
+  { backend: dropbox, options: [], tests: [{
+    hostname: 'content.dropboxapi.com',
+    token: 'x_Dfb5viqakAAAAAAAAGoUwH-_QTKeaKqGoN3iIM-M2capc3jsU0vx_HtZEQdZKu',
```

Comment:
Is this an active token? You should expire it if so, don't want any ol' person getting access to your dropbox. For test tokens, you should:

* Add the token here: https://github.com/superfly/cdn/blob/master/.fly.secrets.yml (don't check your changes in)
* Reference the secret in the config: https://github.com/superfly/cdn/blob/master/.fly.yml#L30

And then use `app.config.dropbox_token` to use it in code, like the S3 test does: https://github.com/superfly/cdn/blob/master/test/backends/aws_s3.spec.ts#L102